test/compile/scale/domain.test.ts

```diff
@@ -474,86 +474,80 @@ describe('compile/scale', () => {
       });
     });
 
-    it('should ignore order: ascending', () => {
+    it('should sort the output domain if one domain is sorted with true', () => {
       const domain = mergeDomains([{
         data: 'foo',
         field: 'a',
-        sort: {field: 'b', op: 'mean', order: 'ascending'}
+        sort: true
       }, {
         data: 'foo',
-        field: 'a',
-        sort: {field: 'b', op: 'mean'}
+        field: 'b',
       }]);
 
-      assert.deepEqual<VgDomain>(domain, {
+      assert.deepEqual(domain, {
         data: 'foo',
-        field: 'a',
-        sort: {field: 'b', op: 'mean'}
+        fields: ['a', 'b'],
+        sort: true
       });
     });
 
-    it('should merge domains with the same data', () => {
+    it('should not sort if no domain is sorted', () => {
       const domain = mergeDomains([{
         data: 'foo',
         field: 'a'
       }, {
         data: 'foo',
-        field: 'b'
+        field: 'b',
       }]);
 
-      assert.deepEqual<VgDomain>(domain, {
+      assert.deepEqual(domain, {
         data: 'foo',
-        fields: ['a', 'b'],
-        sort: true
+        fields: ['a', 'b']
       });
     });
 
-    it('should maintain sort even if false', () => {
+    it('should ignore order: ascending', () => {
```

Comment:
Can you explain why it should ignore ascending?

Reply:
Because it is the default.